content/docs/command-reference/checkout.md

```diff
@@ -145,11 +153,9 @@ baseline-experiment     <- First simple version of the model
 bigrams-experiment      <- Uses bigrams to improve the model
 ```
 
-This project comes with a predefined HTTP
-[remote storage](/doc/command-reference/remote). We can now just run `dvc pull`
-that will fetch and checkout the most recent `model.pkl`, `data.xml`, and other
-files that are tracked by DVC. The model file hash
-`662eb7f64216d9c2c1088d0a5e2c6951` will be used in the `train.dvc`
+We can now just run `dvc pull` that will fetch and checkout the most recent
+`model.pkl`, `data.xml`, and other files that are tracked by DVC. The model file
```

Comment:
@jorgeorpinel can we use `checkout` here instead of `pull` since files tracked by DVC already exist in remote storage. This can be described as `We can now just run dvc checkout that will update the most recent`.

Reply:
Yes! I think that was part of the point of moving fetch into the hidden section 🙂